PSIni/Public/Export-Ini.ps1

```diff
@@ -157,16 +157,37 @@ function Export-Ini {
                 CommentChar           = ";"
                 SkipTrailingEqualSign = $SkipTrailingEqualSign
             }
-            Out-Keys @outKeysParam @fileParameters
+
+            #Collect output from Out-Keys to fileContent
+            $fileContent += Out-Keys @outKeysParam
 
             # TODO: what when the Input is only a simple hash?
 
-            if ($Format -eq "pretty") { Out-File -InputObject "" @fileParameters }
+            if ($Format -eq "pretty") { $fileContent += "" } # Separate Sections with whiteSpace
```

Comment:
@lipkau 
If we don't do newLine fixing (mentioned in another comment), this has an interesting effect of creating an empty line at the end of files we write to.  
I am thinking, if in the end it is good or bad, or against some specification. Because this side effect has this nice property, that appending to a `-pretty` .ini file, we automatically get an empty line between old & new content.

This issue does not consider non `-pretty` .ini files, because they don't add these empty lines, so only one new line is added at the end of the file.